README.md

```diff
@@ -103,6 +103,7 @@ In addition to [the parameters that you can configure for the MySQL Connector/J
 | Parameter       | Value           | Required      | Description  | Default Value |
 | --------------- |:---------------:|:-------------:|:------------ | ------------- |
 |`enableClusterAwareFailover` | Boolean | No | Set to true to enable the fast failover behavior offered by the AWS JDBC Driver. Set to false for simple JDBC connections that do not require fast failover functionality. | `true` |
+|`useConnectionPlugins` | Boolean | No | Use connection plugins to execute JDBC methods. Set to false to disable this feature. **NOTE:** disabling connection plugins will also disable the failover functionality and Enhanced Failure Monitoring. | `true` |
```

Comment:
This shouldn't be here imo, this is more for the manager than plugin

Reply:
Since this parameter would affect the failover functionality I prefer adding this here too.

Reply:
Perhaps it is better to move it outside of this table and add it as a note